.nuget/packages.config

```diff
@@ -3,17 +3,14 @@
   <package id="ILMerge" version="3.0.21" targetFramework="net452" />
   <package id="Microsoft.Build" version="15.1.262-preview5" targetFramework="net45" />
   <package id="Microsoft.VisualStudio.Composition" version="15.8.98" targetFramework="net45" />
-  <package id="Microsoft.VisualStudio.ProjectSystem" version="15.0.582-pre-g76aab6d79c" targetFramework="net45" />
   <package id="Microsoft.VisualStudio.ProjectSystem" version="16.0.201-pre-g7d366164d0" targetFramework="net45" />
-  <package id="Microsoft.VisualStudio.Setup.Configuration.Interop" version="1.1.142-preview4-29265" targetFramework="net35" />
   <package id="Microsoft.VSSDK.BuildTools" version="15.9.3032" targetFramework="net46" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net45" />
   <package id="Lucene.Net" version="3.0.3" targetFramework="net40" />
   <package id="Microsoft.Web.Xdt" version="2.1.2" targetFramework="net40" />
   <package id="NuGet.Core" version="2.14.0-rtm-832" targetFramework="net45"/>
-  <package id="NuGet.MsBuild.Integration" version="3.1.0-beta-001" targetFramework="net45" />
   <package id="VSLangProj140" version="14.0.25029" targetFramework="net46" />
```

Comment:
are we still using this 14.x dependency?

Reply:
Yeah. 
It's just APis that were added in 14.x, but are still relevant for 14.x and later.